### ISSUE TITLE
Rules priority

### ISSUE THREAD

**Issue by jmabad100** [rdmtc/node-red-contrib-sun-position#139]
Good morning,

I have a question about the priority of the rules: 
Is the priority of the rules in relation to their order? Rule 1 runs before rule 2?

I have the following rule settings:

![Captura](https://user-images.githubusercontent.com/47498211/79744670-0f203480-8307-11ea-8c98-a6fd7306025b.JPG)

I have verified that rule 3 is executed before rule 2. In my case, if it rains (rule 3) the blind is positioned at 25%, however, later than Astro Dusk, it should be closed completely due to rule 2.

Can anybody help me again?

Thank so much

**Comment by Hypnos3** [comment]
Hallo,

there is a priority relation, but this is more difficult.
see https://github.com/rdmtc/node-red-contrib-sun-position/blob/master/blind_control.md#rules-execution-in-detail

in other words:
 - to the last rule of type "until" the rules will be evaluated from starting the first rule to the last rule of type until.
- then the rules will be evaluated in reverse order

In your example the order of evaluating rules is following:
- until civil Dawn rule 1 will be evaluated and only rule 1 gets active.
- after civil Dawn the rules will be evaluated from last to first.
  -rule 4
  - then rule 3
  - and then rule 2

**Comment by jmabad100** [comment]
Hallo 

Sorry to ask you again, but I can't get the rules in order to be executed as I want.

Now I have order them as I show you in the screenshot

![Captura de pantalla 2020-04-21 a las 16 35 49](https://user-images.githubusercontent.com/47498211/79878906-461d4580-83ee-11ea-8d44-d1fac6b3436e.png)

and rule 3 and rule 2 are evaluated before than rule 4.(astro dusk)

How should I order them so that Astro Dusk is a higher priority than Rain and Rain&presence

One more time...thank so much

**Comment by Hypnos3** [comment]
Have you try it? In the documentation is a flow for a simulation.

I'm currently due to the situation very busy, so it will take some time until I can have a look into it.

**Comment by Hypnos3** [comment]
You are right this no longer works as expected. 

That was the case, I don't remember why I changed that. I will change that back with the next release.

This is still under test. If you feel brave you can already install it from command line from the node-red directory with (standard Linux):
```bash
npm install node-red-contrib-sun-position@1.1.0-alpha
```

If you are using a Homematic CCU the commands are (to use from ssh login):
```bash
source /usr/local/addons/redmatic/home/.profile
cd /usr/local/addons/redmatic/var
npm install --save --no-package-lock --global-style --save-prefix="~" --production node-red-contrib-sun-position@1.1.0-alpha
```

**Comment by Hypnos3** [comment]
solved with Version 1.1.0
